.travis/before_install/mingw.sh

```diff
@@ -1,11 +1,11 @@
-sudo apt-get install -y mingw32 mingw-w64 libc6-dev-i386 jq texinfo
+sudo apt-get install -y $(test $ARCH == 32 && echo mingw32 libc6-dev-i386 || echo mingw-w64) jq texinfo
```

Comment:
Hehe, very clever, very lispy :+1:

Reply:
a bit hard to read though and presumably only works because of precedence rules.

But fine with me

Reply:
I thought it is a very common thing to do. I agree it's maybe a bit hard to read, I just wanted a quick solution. I think it's fine. It's not that the rest of the Travis stuff is super well written.

Reply:
I've not seen it before. I like it.